recipe/meta.yaml

```
@@ -23,15 +23,16 @@ requirements:
     - wheel
   run:
     - python
-    - more-itertools
     - cheroot >=8.2.1
     - portend >=2.1.1
+    - more-itertools
     - zc.lockfile
     - jaraco.collections
+    - pywin32 >=227   # [win and py<310]
```

Add importlib-metadata for low versions
https://github.com/cherrypy/cherrypy/blob/main/setup.py#L73C10-L73C28
```suggestion
    - pywin32 >=227   # [win and py<310]
    - importlib-metadata # [py<38]
```

---

It's from the main branch. For v18.9.0, it's not needed https://github.com/cherrypy/cherrypy/blob/v18.9.0/setup.py#L68-L72